docker-compose.yaml

```diff
@@ -294,6 +293,25 @@ services:
       - "./.dapr/dapr-config.yaml:/config.yaml"
       - "./.dapr/components:/components"
 
+  media:
+    extends:
+      file: media/docker-compose-base.yaml
+      service: media
+    image: ghcr.io/misarch/media:main
+  media-minio:
+    extends:
+      file: media/docker-compose-base.yaml
+      service: media-minio
+    ports:
+      - 9000:9000
```

Comment:
Do we need this port mapping?

Reply:
Not necessarily if everything this is only accessed in the Docker (or whatever other internal) network